### PR TITLE
Update draft Dropwizard guidance

### PR DESCRIPTION
**Pull request by soupdragon99**
Updated draft guidance to match Rails guidance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,108 +1,138 @@
...
 
-### Maven
-```
-<dependency>
-    <groupdId>engineering.gds-reliability</groupdId>
-    <artifactId>gds-metrics-dropwizard</artifactId>
-    <version>1.0.0</version>
-</dependency>
-```
+    `./gradlew build`
```
Comment (soupdragon99) [inline README.md]: Will someone using this already have the Gradle build tool installed?

File: README.md [hunk 2]
```diff
@@ -1,108 +1,138 @@
...
 
-### Gradle
-```
-implementation 'engineering.gds-reliability:gds-metrics-dropwizard:1.0.0'
-```
+2. Publish to a [Maven][] local repository, by running:
```
Comment (soupdragon99) [inline README.md]: I'm unsure how this step fits with the previous ones.

File: README.md [hunk 3]
```diff
@@ -1,108 +1,138 @@
...
+    ```
+
+    Gradle
+    ```
+    repositories {
+        maven { url "https://dl.bintray.com/reliability-engineering-gds/gds_metrics_dropwizard" }
+    }
+    ```
+
+4. Add your library as a dependency to your project.
```
Comment (soupdragon99) [inline README.md]: I'm unsure how this step fits with the previous ones.

File: README.md [hunk 4]
```diff
@@ -1,108 +1,138 @@
...
-1. Your application must be deployed on [PaaS](https://www.cloud.service.gov.uk/).
-2. Add the environment variable `PROMETHEUS_METRICS_PATH`. Example:
-```
-PROMETHEUS_METRICS_PATH=/metrics
-```
-3. If you want to have Dropwizard extended metrics enabled, define the `ENABLE_DROPWIZARD_METRICS` environment variable. Example:
-```
-ENABLE_DROPWIZARD_METRICS=true
-```
+3. Add the GDS metrics library to where your project is stored by adding:
```
Comment (soupdragon99) [inline README.md]: Where do you add this code?

File: README.md [hunk 5]
```diff
@@ -1,108 +1,138 @@
...
+    implementation 'engineering.gds-reliability:gds-metrics-dropwizard:1.0.0'
+    ```
+
+5. When your app is deployed to the PaaS, you can add an environment variable so Prometheus can discover your app’s metrics, for example:
+
+    ```
+    PROMETHEUS_METRICS_PATH=/metrics
+    ```
+
+## How to Change your project
```
Comment (soupdragon99) [inline README.md]: How is this related to the previous section?

File: README.md [hunk 6]
```diff
@@ -1,108 +1,138 @@
...
+    public void initialize(final Bootstrap<ExampleConfiguration> bootstrap) {
+        ...
+        bootstrap.addBundle(new MetricsBundle());
+        ...
+    }
+    ```
 
-2. Disable app security for the metrics path.
-If your application already has security (authentication) implemented, the metrics path defined in the variable `PROMETHEUS_METRICS_PATH` should be excluded because this path already has its own security. `Configuration.getInstance.getPrometheusMetricsPath()` can be used to recover the proper value.
+2. Disable app security for the metrics path. If your application already has security (authentication) implemented, the metrics path defined in the variable `PROMETHEUS_METRICS_PATH` should be excluded because this path already has its own security. `Configuration.getInstance.getPrometheusMetricsPath()` can be used to recover the proper value.
```
Comment (soupdragon99) [inline README.md]: is this the only type of security authentication? or is it that this is the preferred option? what are we auth'ing? the user to use the app? the app to use the paas?

File: README.md (hunk 1 above)
Comment (philandstuff) [reply README.md]: it doesn't matter, it should Just Work™.  (`./gradlew` calls a gradle wrapper script in this repository, which downloads a local copy of gradle to run)
